Modules/HypnokinkModule/Scenes/HK_RushRepeatConversation.gd

```diff
@@ -0,0 +1,388 @@
+		
+		addButton("Leave", "", "endthescene")
+		
+	if(state == "if_won_serious"):
+		
+		addButton("Heave", "Better get moving", "drag_rush")
+		
+	if(state == "if_lost_serious"):
+		playAnimation(StageScene.Rekt, "kill", {pc="Rush", npc="pc"})
+		saynn("You stumble back, and only have time to wince before Rush throttles your neck and slams you into the wall. Your vision swims.")
+		
+		saynn("[say=Rush]Ya fuckin' stupid-[/say]")
+		
+		saynn("[b]*WHAM*[/b]")
+		
+		saynn("He slams you into the wall again. Through the daze, you wonder if you heard your skull crack.")
+		
+		saynn("[say=Rush]-don't fuckin' [i]ever[/i]-[/say]")
+		
+		saynn("[b]*WHAM*[/b]")
+		
+		saynn("Again, you head makes impact with the metal wall behind you. This time you're sure you heard something important snap, and it begins to dawn on you that you might actually be about to die.")
+		
+		addButton("Oh shit", "Better say something while you still can", "if_lost_serious_2")
+	if(state == "if_lost_serious_2"):
+		playAnimation(StageScene.Rekt, "end", {pc="Rush", npc="pc"})
+		saynn("[say=pc]Wa- wait, I-[/say]")
+		
+		saynn("[say=Rush]SHUT the FUCK up![/say]")
+		
+		saynn("[b]*WHAM*[/b]")
+		
+		saynn("The following impact effectively silences you as you narrowly avoid biting off your tongue. In lieu of speaking again, you cough up blood instead.")
+		
+		saynn("Rush notices your blood on his hand. He lets go with it, rearing up for the punch that ends your life. Then, with the other, he sends you tumbling and skidding across the floor.")
+		
+		saynn("[b]*WHAM*[/b]")
+		
+		saynn("Managing to look up, you see Rush has left a fist shaped imprint in the wall where your head used to be.")
+		
+		saynn("[say=Rush]Don't ever do tha' again. Get th' fuck out.[/say]")
+		
+		saynn("You crawl away before you manage to make Rush lose whatever tenuous grasp on his temper he's managed to regain.")
+		
+		addButton("Ow", "", "endthescene")
+		
+	if(state == "leaving"):
+		playAnimation(StageScene.Duo, "stand", {npc="Rush"})
+		saynn("He does raise a valid point. Or maybe you just don't want your body parts, external [i]or[/i] internal, rearranged. Either way, you leave before he does something you regret.")
+		
+		saynn("[say=Rush]Alrigh', well see ya around pipsqueak! Y'know where t' find me![/say]")
+		
+		saynn("Weirdly enough, he sounds as if he means that as an actual invitation.")
+		
+		addButton("Retreat", "", "endthescene")
+
+
+func _react(_action: String, _args):
+	if(_action == "put_em_up"):
+		runScene("FightScene", ["Rush"], "boxing")
+		return
+		
+	if(_action == "put_em_up_again"):
+		runScene("FightScene", ["Rush", "boxing_serious"], "boxing_serious")
+		return
+		
+	if(_action == "get_fucked_check"):
+		GM.pc.addLust(50)
+		var anus: Orifice = GM.pc.getBodypart(BodypartSlot.Anus).getOrifice()
+		anus.setLooseness(anus.getLooseness() + 1.0)
+		if(anus.getLooseness() > 3.0 and anus.getComfortableInsertion() > 25.0):
```

Comment:
Not a fan of these checks. Dunno, I'd rather you show the scene anyway but also add a `GM.pc.doPainfullyStretchHole(BodypartSlot.Anus, "Rush")` call x3. You can try to change my mind if you think that its important to cockblock (anusblock?) the player here x3

Reply:
My thinking is that like the other branch requiring at least some ability to fight, the sex solution branch should not be a complete freebie (since it does effectively give access to long duration buffs). It also establishes that despite appearances, Rush does care about hurting the player.